providers/fips/fipsprov.c

```diff
@@ -218,7 +218,7 @@ static int fips_get_params_from_core(FIPS_GLOBAL *fgbl)
     * OSSL_PROV_FIPS_PARAM_SECURITY_CHECKS and
     * OSSL_PROV_FIPS_PARAM_TLS1_PRF_EMS_CHECK are not self test parameters.
     */
-    OSSL_PARAM core_params[32], *p = core_params;
+    OSSL_PARAM core_params[28], *p = core_params;
```

Comment:
Its odd, our cross compile targets seem to complain about this being too short an array, but not our non-cross builds.  That doesn't make sense

Reply:
I too wondered why the compiler checks were removed in more recent versions.
My interest is not sufficiently piqued to dive into the gcc change log.

Reply:
obviously my merge didnt go exactly as planned :)